classes/check/collectorcheck.php

```diff
@@ -0,0 +1,78 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Collector failure tests for Check API
+ *
+ * @package admin-tool_cloudmetrics
```

Comment:
This should just be the component name tool_cloudmetrics (and ci should have warned about this....)

Reply:
To see this manually run this check:

php local/moodlecheck/cli/moodlecheck.php -p=admin/tool/cloudmetrics/

This plugin is full of warnings, I'm not sure why they are not being reported on, will chase that